XCUITests/AuthenticationTest.swift

```diff
@@ -96,68 +46,69 @@ class AuthenticationTest: BaseTestCase {
             app.navigationBars["Client.FxAContentView"].buttons["Settings"].tap()
         }
         navigator.nowAt("SettingsScreen")
-        navigator.goto(LoginsSettings)
+        navigator.goto(LockedLoginsSettings)
         waitforExistence(app.navigationBars["Enter Passcode"])
     }
 
     func testPromptPassCodeUponReentryWithDelay() {
-        enablePasscode("133777", interval: "After 5 minutes")
-        navigator.goto(LoginsSettings)
-        waitforExistence(app.navigationBars["Enter Passcode"])
-        typePasscode("133777")
+        navigator.performAction(Action.SetPasscode)
+        setInterval("After 5 minutes")
+        navigator.performAction(Action.UnlockLoginsSettings)
         waitforExistence(app.tables["Login List"])
 
-        //send app to background, and re-enter
+        // Send app to background, and re-enter
         XCUIDevice.shared().press(.home)
         app.activate()
         let contentView = app.navigationBars["Client.FxAContentView"]
         if contentView.exists {
             app.navigationBars["Client.FxAContentView"].buttons["Settings"].tap()
         }
-
         navigator.nowAt("SettingsScreen")
```

Comment:
Nit: `nowAt` is good for documentation here, but: use the constant, rather than the literal string.